otter/integration/config.sh-template

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#
+# To run the integration tests, you'll need to have various environment
+# settings established.
+
+# This is the username used to authenticate against Identity.
+export AS_USERNAME=joe_user
+
+# This is the password used to authenticate against Identity.
+export AS_PASSWORD=joes_password
```

Comment:
Not sure how that helps.  It's checking a dictionary, which means it has to be running inside a Python environment.  But you don't need to be running in a Python environment to accidentally commit passwords to the github repo.  I think that was your original objection.